.sass-lint.yml

```diff
@@ -77,7 +77,9 @@ rules:
   hex-length: 1
   hex-notation: 1
   indentation: 1
-  leading-zero: 1
+  leading-zero:
+    - 1
+    - include: true
```

Comment:
Changes required: Network-Canvas will either need to update to this setting, or stop linting the styles in the UI submodule.

Reply:
Server inherits this lint config, so will need to override or adopt the same style.